src/xdocs/cmdline.xml.vm

```diff
@@ -148,13 +141,21 @@ java -D&lt;property&gt;=&lt;value&gt;  \
         </li>
         <li>
           <code>-b, --branch-matching-xpath xpathQuery</code>
-           - Show Abstract Syntax Tree(AST) branches that match given XPath query.
+           - Shows Abstract Syntax Tree(AST) branches that match given XPath query.
         </li>
         <li>
           <code>-h, --help</code> - print usage help message and exit. Any other option is ignored.
         </li>
         <li>
-          <code>--executeIgnoredModules</code> - Allows ignored modules to be run.
+          <code>-E, --executeIgnoredModules</code> - Allows ignored modules to be run.
+        </li>
+        <li>
+          <code>-C, --checker-threads-number</code> - (experimental) The number of Checker threads
+          (must be greater than zero). Default is 1.
+        </li>
+        <li>
+          <code>-W, --tree-walker-threads-number</code> - (experimental) The number of TreeWalker
```

Comment:
it is sad that we leaked `-C` and `-W` to users, such options do not work.
We should remove them, probably should be done in separate issue.
@rnveach , are you agree ?

Reply:
@romani Shall I do it then?

Reply:
I agree. @gaurabdg Please remove them.

Reply:
@romani @rnveach I am doing the changes in this issue only, because I revert these changes to the master one, then these options will be present in `Main.java` but not in the xdoc, so I have to add extra lines to ignore these in the tests, which would be redundant once these changes are made in master, and I have to change it again.
Or else if I create a new PR, I have to wait for it to merge and then rebase and push changes here.

Reply:
I assume this PR is blocked by https://github.com/checkstyle/checkstyle/pull/8180

Reply:
Yes.

Reply:
This was unblocked and should be rebased if not already.